src/View/AdminUser/AddUserUI.java

```diff
@@ -0,0 +1,151 @@
+package View.AdminUser;
+
+import javax.swing.JPanel;
+import javax.swing.JLabel;
+import javax.swing.JTextField;
+import javax.swing.JCheckBox;
+import javax.swing.JComboBox;
+import javax.swing.JButton;
+import javax.swing.SpringLayout;
+import java.awt.BorderLayout;
+import java.awt.Dimension;
+import javax.swing.BoxLayout;
+import javax.swing.BorderFactory;
+import java.awt.Component;
+
+import Utilities.SpringUtilities;
+import View.Events.UserEvent;
+
+/**
+ * UI to Add User, Form with the fields of user
+ * @author: Lourdes Villca
+ */
+
+public class AddUserUI extends JPanel {
+    private JLabel nameLabel;
+    private JLabel lastNameLabel;
+    private JLabel userLabel;
+    private JLabel passwordLabel;
+    private JLabel activeLabel;
+    private JLabel addressLabel;
+    private JLabel emailLabel;
+    private JLabel roleLabel;
+    private JLabel possitionLabel;
+    private JTextField nameTxt;
+    private JTextField lastNameTxt;
+    private JTextField userTxt;
+    private JTextField passwordTxt;
+    private JCheckBox activeCbox;
+    private JTextField addressTxt;
+    private JTextField emailTxt;
+    private JComboBox roleCbox;
+    private JComboBox possitionCbox;
+    private JButton acceptBtn;
+    private JButton cancelBtn;
+    private UserEvent userEvent;
+    private JPanel leftPanel;
+    private JPanel rightPanel;
+
+    public AddUserUI(){
+
+        leftPanel = makeLeftPanel();
+        rightPanel = makeRightPanel();
+        acceptBtn = new JButton("Accept");
+        cancelBtn = new JButton("Cancel");
+        this.setLayout(new BorderLayout());
+
+        acceptBtn.addActionListener(new UserEvent(this));
+
+        JPanel bottomPanel = new JPanel();
+        bottomPanel.setLayout(new BoxLayout(bottomPanel, BoxLayout.X_AXIS));
+
+        bottomPanel.add(acceptBtn);
+        bottomPanel.add(cancelBtn);
+
+        this.add(leftPanel, BorderLayout.LINE_START);
+        this.add(rightPanel, BorderLayout.LINE_END);
+        this.add(bottomPanel, BorderLayout.PAGE_END);
+
+        this.setBorder(BorderFactory.createTitledBorder("Add New User"));
```

Comment:
anything related to labels and UI text to display should be as part of a constant file or an internazionalization file in case that you need to support multiple languages